x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -79,6 +79,21 @@ const navigationTree: NavigationTreeDefinition = {
                 defaultMessage: 'Anomaly detection',
               }),
               link: 'ml:anomalyDetection',
+              renderAs: 'item',
+              children: [
+                {
```

Comment:
This is just so 'ml:anomalyDetection' is active in the side nav when any of the pages below are active